src/HMI/tmplits.css

```diff
@@ -132,4 +132,55 @@ input::-webkit-outer-spin-button,
 input::-webkit-inner-spin-button {
   -webkit-appearance: none;
   margin: 0;
+}
+
+.tmplit-table {
+    width: 100%;
+    display: grid;
+    padding: 0px 20px 20px 20px;
+    overflow: auto;
+    grid-template-columns: var(--tmplit-table-grid-template-columns, 1fr 4fr 1fr);
+    text-align: var(--tmplit-table-text-align, left);
+    grid-template-rows: var(--tmplit-table-grid-template-rows, repeat(auto-fill, 50px));
+    height: var(--tmplit-table-height, 470px);
+    background-color: var(--tmplit-table-background-color, var(--md-sys-color-surface-container, #FFFFFF));
+}
+
+.tmplit-table-header {
+    font-size: var(--tmplit-table-header-font-size, 24px);
+    font-weight: var(--tmplit-table-header-font-weight, 700);
+    padding: var(--tmplit-table-header-padding, 20px 20px 10px 20px);
+    text-align: var(--tmplit-table-header-text-align, center);
+    color: var(--tmplit-table-header-color, var(--md-sys-color-on-surface, black));
+    background-color: var(--tmplit-table-header-background-color, var(--md-sys-color-surface-container, #FFFFFF));
+}
+
+.tmplit-table-row {
+    display: grid;
+    grid-column: var(--tmplit-table-row-grid-column, 1 /10);
+    grid-template-columns: subgrid;
+    padding: 1px;
+    align-items: center;
+    border-style: var(--tmplit-table-row-border-style, none none solid none); 
+    border-width: var(--tmplit-table-row-border-width, 1px);
+    font-size: var(--tmplit-table-row-font-size, 18px);
+    border-color: var(--tmplit-table-row-border-color, var(--md-sys-color-outline-variant, gray));
```

Comment:
Falling back to the material tokens is a great idea. Nice.

Reply:
yeah josh and i saw default material buttons do that so we copied them.